server/app/controllers/article_category_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ArticleCategoryController < ApplicationController
+
+      # POST /article_category 
+      def create
+        current_user = User.find_by(session[:user_id])
+        if current_user
+            article_cat = ArticleCategory.create(article_category_params)
+            if article_cat.save
```

Comment:
Once you create an item in the db with `create` method, you do not need to use `save`, you can just check for validity using `valid?`.
Creating then saving performs two operations on the db as opposed to one with create and checking validity.